hilgendorf_moveit_config/launch/default_warehouse_db.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
```

Comment:
So currently the only file that calls default_warehouse_db.launch is demo.launch. That inclusion is conditional on the user passing the db:=true argument and in that case they can specify the location of the database they want to use.

Can you point out where the problem is? Should I get rid of the default db location in case this ever gets installed to a non-writable location?